.rubocop_fixme.yml

```diff
@@ -23,7 +23,7 @@ Metrics/AbcSize:
 
 Metrics/ModuleLength:
   Exclude:
-    - 'app/controllers/concerns/hyrax/collections_controller_behavior.rb'
+    - 'app/controllers/concerns/hyrax/dashboard/collections_controller_behavior.rb'
```

Comment:
@jcoyne is correct that in the past I argued for referring to anything a user can do administratively (that is, things that involved managing objects) as "administration" and thus the logical path would include `/admin`.

However, I think @mjgiarlo made a suggestion more than once (not sure whether it was in Slack, Github tickets, or both, and hope I'm not misrepresenting you, @mjgiarlo!) to use "dashboard" to refer to what I was calling "administration," I believe to remove the potential confusion over what a repository administrator can do and what a user with edit privileges (for her own works, for example) can do, since some people seem to consider only what a repository administrator can do as "administration."

So I personally prefer "administration" (see my last paragraph below for more explanation of why) but I don't think my preference is strong enough to counter the argument for "dashboard" if @mjgiarlo and most people in the community think that is a more intuitive term to use. That's why I didn't object to switching to "dashboard." I don't want to stand in the way if the general consensus is that "dashboard" makes more sense.

I am a little worried about trying to use both terms, as outlined in the quoted text in @elrayle's comment above, because repository administrators can also manage another user's work, but using that scheme that action would come under `/dashboard` not `/admin` even though other things the repository admin does (e.g., manage users) would come under `/admin`. The line seems blurry so using both paths might be confusing.

The reason I prefer "administration" to refer to anything the user does on the non-public side is because the term "dashboard" in interaction design has had a more specific meaning, to refer to quick displays of critical information about a website or application:

> "The term “dashboard” is a metaphor for a car dashboard, which provides essential information that the driver can absorb quickly, without having to think." 
[link](https://www.nngroup.com/articles/dashboards-preattentive/)

Our "Activity Summary" page is what has more commonly been called a "dashboard." So calling the whole admin side a dashboard conflates the specific dashboard page with all the other administrative features. That's what I don't feel comfortable with. On the other hand, in recent years I think it has become somewhat more common to refer to user administration UIs as the "dashboard." So I don't really like it, but I also accept that the term has possibly evolved a more general meaning and thus don't feel like I can make too strong an argument based on the traditional meaning of a "dashboard."